RawBencher/packages.config

```diff
@@ -40,7 +40,7 @@
   <package id="NPoco" version="3.9.4" targetFramework="net462" />
   <package id="Remotion.Linq" version="2.2.0" targetFramework="net462" />
   <package id="Remotion.Linq.EagerFetching" version="2.2.0" targetFramework="net462" />
-  <package id="RepoDb" version="1.8.0" targetFramework="net462" />
+  <package id="RepoDb" version="1.8.2" targetFramework="net472" />
```

Comment:
Hi Frans, my local was on (net472). But this will still work, or else, please let me know if you want me to change this one again. Thanks